toolchain/language_server/server.h

```diff
@@ -38,46 +39,48 @@ class LanguageServer : public clang::clangd::Transport::MessageHandler,
   // LSPBinder::RawOutgoing
 
   // Send method call to client
-  void callMethod(llvm::StringRef method, llvm::json::Value params,
-                  clang::clangd::Callback<llvm::json::Value> reply) override {
+  auto callMethod(llvm::StringRef method, llvm::json::Value params,
+                  clang::clangd::Callback<llvm::json::Value> reply)
+      -> void override {
     // TODO: implement when needed
   }
 
   // Send notification to client
-  void notify(llvm::StringRef method, llvm::json::Value params) override {
+  auto notify(llvm::StringRef method, llvm::json::Value params)
+      -> void override {
     transport_->notify(method, params);
   }
 
  private:
-  const std::unique_ptr<clang::clangd::Transport> transport_;
-  // content of files managed by the language client.
-  std::unordered_map<std::string, std::string> files_;
-  // handlers for client methods and notifications
-  clang::clangd::LSPBinder::RawHandlers handlers_;
-
-  explicit LanguageServer(std::unique_ptr<clang::clangd::Transport> transport)
-      : transport_(std::move(transport)) {}
-
   // Typed handlers for notifications and method calls by client.
 
   // Client opened a document.
-  void OnDidOpenTextDocument(
-      clang::clangd::DidOpenTextDocumentParams const& params);
+  auto OnDidOpenTextDocument(
+      clang::clangd::DidOpenTextDocumentParams const& params) -> void;
 
   // Client updated content of a document.
-  void OnDidChangeTextDocument(
-      clang::clangd::DidChangeTextDocumentParams const& params);
+  auto OnDidChangeTextDocument(
+      clang::clangd::DidChangeTextDocumentParams const& params) -> void;
 
   // Capabilities negotiation
-  void OnInitialize(clang::clangd::NoParams const& client_capabilities,
-                    clang::clangd::Callback<llvm::json::Object> cb);
+  auto OnInitialize(clang::clangd::NoParams const& client_capabilities,
+                    clang::clangd::Callback<llvm::json::Object> cb) -> void;
 
   // Code outline
-  void OnDocumentSymbol(
+  auto OnDocumentSymbol(
       clang::clangd::DocumentSymbolParams const& params,
-      clang::clangd::Callback<std::vector<clang::clangd::DocumentSymbol>> cb);
+      clang::clangd::Callback<std::vector<clang::clangd::DocumentSymbol>> cb)
```

Comment:
Do we need to include `<vector>` for this?

Reply:
Added includes